mythical-creatures/spec/medusa_spec.rb

```diff
@@ -25,17 +25,40 @@
   it 'turns a person to stone when staring at them' do
     medusa = Medusa.new('Cassiopeia')
     victim = Person.new('Perseus')
-
-    assert(victim.stoned?).to be false
+    
+    expect(victim.stoned?).to be false
     medusa.stare(victim)
-    assert(victim.stoned?).to be true
+    expect(victim.stoned?).to be true
   end
-
+  
   it 'can only have three victims' do
     # your code here
-  end
+    medusa = Medusa.new('Cassiopeia')
+    victim = Person.new('Perseus')
+    victim1 = Person.new('Perseus1')
+    victim2 = Person.new('Perseus2')
 
+    medusa.stare(victim)
+    medusa.stare(victim1)
+    medusa.stare(victim2)
+    
+    expect(medusa.capped?).to be true
```

Comment:
This is a good start! One caveat to testing whether Medusa is capped, however, is that we still don't know if she can only have 3 victims.

It would be great to assert the "victim" behavior itself. For example, when she stares at a person, we know they become a victim because they are stoned. What happens when there is a fourth person? We would expect that they do not become stoned after she has stared at them.

The following test will confirm that behavior:

```
medusa = Medusa.new('Cassiopeia')
victim1 = Person.new('Perseus1')
victim2 = Person.new('Perseus2')
victim3 = Person.new('Perseus3')
victim4 = Person.new('Perseus4')

medusa.stare(victim1)
expect(victim1.stoned?).to be true
medusa.stare(victim2)
expect(victim2.stoned?).to be true
medusa.stare(victim3)
expect(victim3.stoned?).to be true
medusa.stare(victim4)
expect(victim4.stoned?).to be false
```